docs/reference/release-notes/7.10.asciidoc

```diff
@@ -3,6 +3,18 @@
 
 Also see <<breaking-changes-7.10,Breaking changes in 7.10>>.
 
+[float]
+[[security-updates-7.9.0]]
+=== Security updates
+
+* An information disclosure flaw was found in the Elasticsearch async search API.
+Users who execute an async search will store the HTTP headers.
+An Elasticsearch user with the ability to read the .tasks index could obtain
+sensitive request headers of other users in the cluster.
+All versions of {es} between 7.7.0 and 7.10.1 are affected by this flaw.
+You must upgrade to {es} version 7.10.2 to obtain the fix.
+https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-22132[CVE-2021-22132]
```

Comment:
Just a few minor nits: Removes some extraneous mentions of Elasticsearch and formats the `.task` index name.
```suggestion
* An information disclosure flaw was found in the {es} async search API.
Users who execute an async search will store the HTTP headers.
A user with the ability to read the `.tasks` index could obtain
sensitive request headers of other users in the cluster.
All versions of {es} between 7.7.0 and 7.10.1 are affected by this flaw.
You must upgrade to {es} version 7.10.2 to obtain the fix.
https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-22132[CVE-2021-22132]
```